### ISSUE TITLE
NormalizeWhitespace() does not indent switch expression syntax properly

### ISSUE THREAD

**Issue by mahalex** [dotnet/roslyn#45651]
**Version Used**: 

3.7.0-3.final

**Steps to Reproduce**:
```
var node = SyntaxFactory.ParseStatement("return a switch {0=>2,1=>1,_=>0};");
var actual = node.NormalizeWhitespace("  ").ToFullString();
```

**Expected Behavior**:
`actual` should be
```
return a switch
{
  0 => 2,
  1 => 1,
  _ => 0
};
```

**Actual Behavior**:
`actual` is
```
return a switch
{
0 => 2, 1 => 1, _ => 0
}

;
```
**Comments**:

there are two issues:

- switch expression arms are not indented correctly;

- unnecessary line breaks after close brace are inserted.

**Comment by trejjam** [comment]
Hello @jaredpar, are there any plans here?

**Comment by svick** [comment]
This is a duplicate of https://github.com/dotnet/roslyn/issues/52543 and is fixed in Roslyn 4.0, which should be released very soon.
